src/bundle/Resources/public/scss/_tables.scss

```diff
@@ -187,7 +187,7 @@
 }
 
 .ibexa-dashboard {
```

Comment:
I would do it either on modifier or move it to dashboard css, so that _tables.scss could be pure